compiler/luci-interpreter/src/kernels/FullyConnected.test.cpp

```diff
@@ -204,6 +204,106 @@ TEST(FullyConnectedTest, SimpleS4)
               FloatArrayNear(output_data, quantized_tolerance));
 }
 
+TEST(FullyConnectedTest, SimpleU4PerTensor)
```

Comment:
Can we add some negative tests too?
Our SE requirement is to keep num(pos) <= num(neg)